build-system/tasks/extension-generator/bento/amp-__component_name_hyphenated__/amp-__component_name_hyphenated__.md

```diff
@@ -0,0 +1,30 @@
+---
+$category@: presentation
+formats:
+  - websites
+teaser:
+  text: Teaser text description of component.
+experimental: true
+---
+
+<!---
+Copyright __current_year__ The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# amp-__component_name_hyphenated__
+
+## Usage
+
+Describe how the component is used; see other extensions for examples of what content to include.
```

Comment:
This should probably be the same as `build-system/tasks/extension-generator/extension-doc.template.md`

Maybe modify its format to match this one, so that the file can be shared.

Reply:
Addressed in https://github.com/ampproject/amphtml/pull/31965
Went with duplication to keep logic for copying templates simple/consistent, since this file doesn't change much. PTAL